### ISSUE TITLE
platform e-mail facility

### ISSUE THREAD

**Issue by dimitriirtimid** [local-motion/product#127]
The platform should be able to send e-mails. This will enable notifications and rich email content.

DoD:
- [ ] Send mails from noreply@<platform hostname>
- [ ] Able to send html formatted e-mails

**Comment by jamalv** [comment]
As you've already done this for forus @pantyukhov, could you look into this as your first issue on Localmotion? Make a proposal post it on slack. Maybe you can re-use code / fork from https://github.com/teamforus/forus-notification

**Comment by maxvisser** [comment]
I think this one is safe to work on as it has no direct impact on any strategy we take.

**Comment by jamalv** [comment]
related: https://github.com/local-motion/product/issues/7

**Comment by jknnr** [comment]
@dimitriirtimid Got a question. The project is described as micro-services system, but currently the code is monolithic. This effects the decision concerning how to build the notification service. Could you provide some more information about what we should stick to?

**Comment by maxvisser** [comment]
I think we should still spec it a bit out before starting on it; maybe @AnandAili can write some ideas down.
